packages/internals/src/wasm.ts

```diff
@@ -1,14 +1,14 @@
-import prismaFmt from '@prisma/prisma-fmt-wasm'
+import prismaSchemaWasm from '@prisma/prisma-schema-wasm'
 
 import { WasmPanicRegistry } from './WasmPanicRegistry'
 
 // Note: using `import { dependencies } from '../package.json'` here would break esbuild with seemingly unrelated errors.
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { dependencies } = require('../package.json')
 
-export { prismaFmt }
+export { prismaSchemaWasm }
 // e.g. 4.3.0-18.a39215673171b87177b86233206a5d65f2558857
-export const prismaFmtVersion: string = dependencies['@prisma/prisma-fmt-wasm']
+export const prismaSchemaVersion: string = dependencies['@prisma/prisma-schema-wasm']
```

Comment:
Shouldn't this be `prismaSchemaWasmVersion` then?

Reply:
Right, https://github.com/prisma/prisma/pull/20106